docs/reference/sql/functions/geo.asciidoc

```diff
@@ -5,6 +5,18 @@
 
 The geo functions work with geometries stored in `geo_point` and `geo_shape` fields, or returned by other geo functions.
 
+==== Limitations
```

Comment:
We have a Limitations page where various bits and pieces are listed in a central place: https://www.elastic.co/guide/en/elasticsearch/reference/master/sql-limitations.html
It's probably ok to list this limitation here, but I would like to see it listed in the general Limitations page, as well.